jupyter/environment-r-omero.yml

```diff
@@ -0,0 +1,23 @@
+channels:
+- bioconda
+- conda-forge
+- defaults
+dependencies:
+- bioconductor-ebimage
```

Comment:
Add a note that this library  is not needed for r-gateway to work
This is a library used to manipulate pixels data in an example